src/app/helpers/filters/audioRecordingFilters.ts

```diff
@@ -0,0 +1,78 @@
+import { DateExpressions, InnerFilter } from "@baw-api/baw-api.service";
+import { DateTime } from "luxon";
+import { AudioRecording } from "@models/AudioRecording";
+import { addFilter } from "./filters";
+
+/**
+ * Adds date range conditions to an existing filter
+ *
+ * @param startDate (optional) The minimum `recordedDate` of the date range
+ * @param endDate (optional) The maximum `recordedEndDate` allowed
+ * @param filters (optional) An existing filter to add the date range conditions to. If no filter is provided, a new filter will be created
+ * @returns A new filter with all the same conditions as the initial filter, with the date range conditions added
+ */
+export function addDateFilters(
+  startDate?: DateTime,
+  endDate?: DateTime,
+  filters?: InnerFilter
+): InnerFilter {
+  if (startDate) {
+    // the baw-api is expecting the date in the format (YYYY-MM-DD)
+    // ISO date format is used as it emits the date in the correct format and explicitly emits the time as 00:00:00
+    const formattedStartDate = startDate.toISO();
```

Comment:
I'm surprised you have to do this; luxon DateTime implements `toJSON` itself which emits ISO8601 dates... 
![image](https://user-images.githubusercontent.com/1452968/231669618-2eed0ded-326c-461a-91b8-76de3f4ace9b.png)